include/classes/BasicChecks.class.php

```diff
@@ -444,8 +444,8 @@ public static function valid_lang_code($code)
 		else if (strlen($code) == 3) $sql .= "code_3letters = '".$code ."'";
 		else return false;
 		
-		$result	= mysql_query($sql, $db) or die(mysql_error());
-		$row = mysql_fetch_assoc($result);
+		$result	= mysqli_query($db, $sql) or die(mysqli_error($db));
```

Comment:
The reference of `global $db;` will not work since `$db` is only accessible in DAO classes. After a search thru the code base, it seems this function is not in use at all. If so, this function can be removed. Please confirm. Thanks.